tests/api2/test_420_smb.py

```diff
@@ -12,13 +12,14 @@
 from time import sleep
 apifolder = os.getcwd()
 sys.path.append(apifolder)
-from functions import PUT, POST, GET, DELETE, SSH_TEST
+from functions import PUT, SSH_TEST
```

Comment:
There are only 4 more instances of using the `PUT` REST interface, why not remove them all since you're doing it mostly everywhere with these changes?